app/experimenter/nimbus-ui/src/components/HeaderExperiment/index.tsx

```diff
@@ -7,31 +7,62 @@ import React from "react";
 import { getExperiment_experimentBySlug } from "../../types/getExperiment";
 import { StatusCheck } from "../../lib/experiment";
 import "./index.scss";
+import { humanDate, stringDateSubtract } from "../../lib/visualization/utils";
 
 type HeaderExperimentProps = Pick<
   getExperiment_experimentBySlug,
-  "name" | "slug"
-> & { status: StatusCheck };
+  "name" | "slug" | "startDate" | "endDate"
+> & { status: StatusCheck; analysisRequired?: boolean };
 
-const HeaderExperiment = ({ name, slug, status }: HeaderExperimentProps) => (
-  <header className="border-bottom" data-testid="header-experiment">
-    <h1 className="h5 font-weight-normal" data-testid="header-experiment-name">
-      {name}
-    </h1>
-    <p
-      className="text-monospace text-secondary mb-1 small"
-      data-testid="header-experiment-slug"
-    >
-      {slug}
-    </p>
-    <p className="header-experiment-status position-relative mt-2 d-inline-block">
-      <StatusPill label="Draft" active={status.draft} />
-      <StatusPill label="Review" active={status.review} />
-      <StatusPill label="Live" active={status.live} />
-      <StatusPill label="Complete" active={status.complete} padded={false} />
-    </p>
-  </header>
-);
+const HeaderExperiment = ({
+  name,
+  slug,
+  startDate = "",
+  endDate = "",
+  analysisRequired = false,
+  status,
+}: HeaderExperimentProps) => {
+  return (
+    <header className="border-bottom" data-testid="header-experiment">
+      <div className="row">
+        <div className="col-md-6">
+          <h1
+            className="h5 font-weight-normal"
+            data-testid="header-experiment-name"
+          >
+            {name}
+          </h1>
+          <p
+            className="text-monospace text-secondary mb-1 small"
+            data-testid="header-experiment-slug"
+          >
+            {slug}
+          </p>
+          <p className="header-experiment-status position-relative mt-2 d-inline-block">
+            <StatusPill label="Draft" active={status.draft} />
+            <StatusPill label="Review" active={status.review} />
+            <StatusPill label="Live" active={status.live} />
+            <StatusPill
+              label="Complete"
+              active={status.complete}
+              padded={false}
+            />
+          </p>
+        </div>
+        {analysisRequired ? (
+          <div className="text-right col-md-6">
+            <span className="font-weight-bold">{humanDate(startDate!)} </span>
+            to
+            <span className="font-weight-bold"> {humanDate(endDate!)} </span>(
+            {stringDateSubtract(endDate!, startDate!)} days)
+          </div>
+        ) : (
+          ""
+        )}
+      </div>
+    </header>
+  );
+};
```

Comment:
An alternative layout might be to instead wrap the status container in the row/col setup so the dates are fixed to the bottom of the header:

![Screen Shot 2020-12-16 at 12 00 20 PM](https://user-images.githubusercontent.com/6392049/102380754-4c23f000-3f96-11eb-94f4-c0f231879bf8.png)

A benefit of this would be that the header title and slug are variable in length and could expand as much as needed, whereas the status badges never vary width. It also helps maintain the title hierarchy.

Purely optional 😸

Reply:
I'm not sure I follow what you mean by "wrap the status container in the row/col setup". Could you show it roughly in jsx? Thanks! =)

Reply:
@emtwo sure! Here's what I meant: https://gist.github.com/jodyheavener/bc57dbfd0f0bb66235ffc27c945865e7

Reply:
Also FYI I am working on a issue that updates the `humanDate` utility to only show the year if it's not the current year.

Reply:
Thanks!